pulp_puppet_common/pulp_puppet/common/sync_progress.py

```diff
@@ -43,8 +59,9 @@ def from_progress_dict(cls, report):
 
         :param report: progress report retrieved from the server's task
         :type  report: dict
+
         :return: instance populated with the state in the report
-        :rtype:  SyncProgressReport
+        :rtype:  cls
```

Comment:
`SyncProgressReport` does seem more straightforward, but if subclassed it would not return `SyncProgressReport` on the inherited method. For that reason I'll leave this one as-is.